toolbox/fdc3-workbench/src/components/Header.tsx

```diff
@@ -79,7 +79,7 @@ export const Header = (props: { fdc3Available: boolean }) => {
 				<Toolbar className={classes.toolbar}>
 					<div>
 						<Typography variant="h3" color="inherit" className={classes.fdc3}>
-							<img src="./fdc3-logo.svg" className={classes.headerCube} />
+							<img src={`${import.meta.env.BASE_URL}/fdc3-logo.svg`} className={classes.headerCube} />
```

Comment:
might get away with just `fdc3-logo.svg` here... although that requires the URL used to access to have a trailing / - I've got another hacky fix that just puts that / on in the index page if its not there...